github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1881,6 +1881,35 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
+    annotations:
+      fork-per-release: "true"
+    cluster: prow-workloads
```

Comment:
@rhrazdil here it should be `cluster: ibm-prow-jobs`

Reply:
[Updated](https://github.com/kubevirt/project-infra/compare/fbabfb67e773c6b1d7750ebaec719c181c69afbd..bcbd7937925cb334db92c2e5607a960ba17d1049)